module4/assessments/Week1-3021.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+title: Week 1 Assessment
+---
+
+## Questions and Exercises (10 Points Total)
+
+**Fork** [this repl](https://replit.com/@launch-team/Mod-4-Week-1-Assessment) and complete the questions and exercise it contains.  
+
+
+<aside class="instructor-notes" markdown="1">
+    <p><strong>Instructor Note</strong><br>Here is one solution: https://replit.com/@launch-team/Mod-4-Week-1-Assessment-Solution</p>
+</aside>
+
+### Submission and Feedback
+When completed, click the `Invite` button in the upper right corner and `Generate a join link`.  Include that link in your form submission.
+
+![](/assets/images/module1/Week1/GenerateJoinLink.png)
+
+## Part 1: Questions (10 Points Total)
+
+Answer the questions in the `questions.md` file in the Replit.
+
+## Part 2: Build a Quiz Question (10 Points Total)
+
+Your goal is to build off of the starter code to replicate the behavior in the following video.
+<video controls width="800">
+    <source src="https://github.com/turingschool/launch-curriculum/assets/11747682/80e470c0-611d-4be9-aaed-f8b7f8dea095" type="video/mp4">
+    Your browser does not support the video tag.
+</video>

```

Comment:
The video doesn't show up here but I grabbed the link from the `<video>` tag. I think I paid more attention to your bookmarks to make sure you didn't display anything untoward LOL.

I also notice you included the `.mp4` object but don't use it. Are they the same video?

Reply:
Good call on the .mp4 video, that was from my first attempt at including a video file before I decided to host it on Github, which seems to work well. Will remove the old file.

Thanks for the second set of eyes on the bookmarks, but they should all be bland launch stuff. lol.